projects/exui-common-lib/src/lib/components/search-location/search-location.component.html

```diff
@@ -1,44 +1,24 @@
-<div class="govuk-form-group">
-    <div class="location-picker-custom">
-        <label id="input-selected-location-label" class="govuk-body" for="input-selected-location">
-            Search for a location by name
-        </label>
-        <br>
-        <div class="auto-complete-container">
-            <div>
-                <div class="autocomplete__wrapper">                
-                        <input 
-                            [formControl]="findLocationForm.controls.findLocationControl"
-                            [matAutocomplete]="auto"
-                            aria-expanded="false" 
-                            aria-owns="input-selected-location__listbox" 
-                            aria-autocomplete="both" 
-                            aria-describedby="input-selected-location-label" 
-                            autocomplete="off" 
-                            class="autocomplete__input autocomplete__input--default search-box"
-                            [attr.disabled]="disabled"
-                            [value]="selectedLocation?.court_name"
-                            id="input-selected-location">
-                        <mat-autocomplete class="mat-autocomplete-panel-extend" autoActiveFirstOption #auto="matAutocomplete">
-                            <mat-option *ngFor="let location of locations$ | async"
-                                        [value]="getDisplayName(location)"
-                                        (onSelectionChange)="onSelectionChange(location)"
-                                        [ngClass]="{'hide-autocomplete': !showAutocomplete}">
-                                {{ getDisplayName(location) }}
-                            </mat-option>
-                        </mat-autocomplete>
-                </div>
-            </div>
-        </div>
-        <a href="javascript:void(0)" (click)="addSelection()" class="govuk-button govuk-button--secondary" data-module="govuk-button">
-            Add location
-        </a>
-    </div>
-    <ul  *ngIf="(selectedLocations$ | async)?.length" class="hmcts-filter-tags selection-container">
-        <li class="location-selection"  *ngFor="let selection of selectedLocations$ | async">
-            <a class="hmcts-filter__tag" (click)="removeSelection(selection)" href="javascript:void(0)">           
-                {{ selection.court_name }}
-            </a>
-        </li>
-    </ul>
+<div class="auto-complete-container">
+  <div class="autocomplete__wrapper">                
+    <input                             
+      [formControl]="findLocationFormGroup.controls.findLocationFormControl"
+      [matAutocomplete]="auto"
+      (keyup)="onKeyUp($event)"
+      aria-expanded="false" 
+      aria-owns="input-selected-location__listbox" 
+      aria-autocomplete="both" 
+      autocomplete="off" 
+      class="autocomplete__input autocomplete__input--default search-box"
+      [attr.disabled]="!disabled ? null: disabled"
```

Comment:
is this needed to be reverted?

Reply:
Yes in case the user off the property puts false in